python_src/src/lamp_py/performance_manager/l1_rt_metrics.py

```diff
@@ -148,10 +147,10 @@ def process_metrics_table(
                         ).over(
                             partition_by=(
                                 trips_for_metrics.c.parent_station,
-                                trips_for_metrics.c.route_id,
+                                trips_for_metrics.c.branch_route_id,
                                 trips_for_metrics.c.direction_id,
                             ),
-                            order_by=trips_for_metrics.c.sort_timestamp,
+                            order_by=trips_for_metrics.c.stop_timestamp,
```

Comment:
since we're subtracting `stop_timestamp` from `lag stop_timestamp` we can just sort on `stop_timestamp`